benchmarks/run-benchmark.sh

```diff
@@ -3,12 +3,47 @@
 # Runs a benchmark using drun. The script assumes drun is available on the caller's path.
 set -euo pipefail
 
+# Remove downloaded didc, drun.
+trap 'set +e && rm drun didc && set -e' EXIT SIGINT
+
 BENCH_NAME=$1
 FILE=$(mktemp)
 
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Add script directory to the PATH
+export PATH="$SCRIPT_DIR:$PATH"
+
+get_didc_release(){
+  OS=$(uname)
+  ARCH=$(uname -m)
+  if [ "$OS" == "Darwin" ] && [ "$ARCH" == "arm64" ]; then
+    # Apple sillicon
+    wget -O didc https://github.com/dfinity/candid/releases/download/2023-07-25/didc-macos
+  elif [ "$OS" == "Linux" ] && [ "$ARCH" == "x86_64" ]; then
+          # Linux x86 64bit
+          wget -O didc https://github.com/dfinity/candid/releases/download/2023-07-25/didc-linux64
+      else
+          echo "Unsoported machine"
+          EXIT SIGINT
+  fi
+}
+
+get_drun_release() {
+  OS=$(uname | tr '[:upper:]' '[:lower:]')
+  URL="https://github.com/dfinity/ic/releases/download/release-2023-09-27_23-01%2Bquic/drun-x86_64-${OS}.gz"
+  wget -O "drun.gz" "${URL}"
+  gzip -d drun.gz
+}
+
+if ! type "didc" > /dev/null; then
```

Comment:
Let's please verify the checksum as discussed

Reply:
Done.

Reply:
Why do we need checksum for didc?

Reply:
In the case of drun it is obvious why we need the same release. But in the case of didc I do not see any benefit of it. I am not understanding the second part of the comment.

Reply:
Done, PTAL.